ietf-layer0-types-ext@2021-02-12.yang

```diff
@@ -0,0 +1,853 @@
+module ietf-layer0-types-ext {
```

Comment:
I think that this file should not be added here.

Reply:
that was a shortcut to permit compilation

Reply:
Fixed with https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/53/commits/6be7c2d6b88b6a6ce5d5d99ab7be2396238e77ba